docker/Dockerfile_txsim

```diff
@@ -0,0 +1,48 @@
+# stage 1 Generate celestia-appd Binary
```

Comment:
[nit] I don't understand the rationale for the casing in this comment

```suggestion
# Stage 1: generate celestia-appd binary
```

Reply:
It's a copy-cat from Dockerfile and Dockerfile.ephemeral. 
I guess a follow-up PR will fix other Dockerfiles. 

Editing this one, tho

Reply:
#2117 to track